templates/workflows-kms-key.yaml

```diff
@@ -8,7 +8,15 @@ Parameters:
     Description: URL of S3 bucket where templates are deployed
     ConstraintDescription: Must be a valid S3 HTTP URL
 
+Mappings:
+  Accounts:
+    "035458030717":
+      AdminRole: "arn:aws:iam::035458030717:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_580e9f32ac55c4e7"
+    "728882028485":
+      AdminRole: "arn:aws:iam::728882028485:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_3a9d61d0884be260"
```

Comment:
There is only one set of SSO user accounts and roles.  All SSO user account and roles used to access nextflow accounts (dev & prod) are in the our AWS organizations account (531805629419) therefore this should be.. 

```
    "035458030717":
      AdminRole: "arn:aws:iam::531805629419:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_580e9f32ac55c4e7"
    "728882028485":
      AdminRole: "arn:aws:iam::531805629419:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_3a9d61d0884be260"
```

Also Are you sure you want admins in the nextflow dev account to have access to keys that will unlock data in your nextflow prod account?  That's probably not a good idea, just want to to make sure because I think this is what's going to happen here.

Reply:
Oh, thanks for the correct of the account to use in the role ARNs, as I hadn't noticed.

Re "Are you sure you want admins in the nextflow dev account to have access to keys that will unlock data in your nextflow prod account" <-- that makes no sense since we're using `FindInMap` based on account id.

Reply:
please ignore my comment, `There is only one set of SSO user accounts and roles`. it is not correct. there is one set of users but i forgot that SSO duplicates roles to each account that users access therefore your map is correct